edx_proctoring/statuses.py

```diff
@@ -235,3 +235,42 @@ def validate(cls, status):
             )
             raise ProctoredExamBadReviewStatus(err_msg)
         return True
+
+
+class InstructorDashboardOnboardingAttemptStatus:
+    """
+    A class to enumerate the different statuses a proctored exam attempt
+    in an onboarding exam can have and to map them to internal database statutes.
+    These are intended to be used in externally facing applications, such
+    as the Instructor Dashboard.
+    """
+    not_started = 'not_started'
+    started = 'started'
+    submitted = 'submitted'
+    rejected = 'rejected'
+    verified = 'verified'
+    error = 'error'
+
+    onboarding_statuses = {
+        ProctoredExamStudentAttemptStatus.created: started,
+        ProctoredExamStudentAttemptStatus.download_software_clicked: started,
+        ProctoredExamStudentAttemptStatus.ready_to_start: started,
```

Comment:
This nomenclature seems like it may cause confusion; can we switch these to correspond to `not_started` instead?

Reply:
I think `not_started` really means `no attempt has been created`. Started here do collide with the exam start. Maybe we can have two started? Like `setup started` vs `proctoring started`? This way, we can avoid the collision with `started` status in database.

Reply:
That makes sense to me. I'll make that change.